### PR TITLE
pt_BR locale update

### PR DESCRIPTION
**Pull request by mtub**
Updated pt_BR locale files based on locale files provided by Ramón (see http://forum.pkp.sfu.ca/t/how-to-submit-translation-for-review/9033 and https://pkp.sfu.ca/wiki/index.php?title=OJS:_Portuguese_%28pt_BR%29)

### CODE REVIEW

**Comment by mtub** [comment]
@asmecher , can you take a quick look?

File: plugins/blocks/customnavbar/locale/pt_BR/locale.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/blocks/customnavbar/locale/pt_BR/locale.xml
+  *
+  * Copyright (c) 2013-2014 Simon Fraser University Library
```
Comment (asmecher) [inline plugins/blocks/customnavbar/locale/pt_BR/locale.xml]: Old copyright dates. (Also below)

**Comment by mtub** [comment]
Removed the locale files with wrong copyright date - they didn't belong in here. While on the task, fixed another wrong copyright date and removed another orphaned locale file.

File: plugins/importexport/crossref/locale/pt_BR/locale.xml [hunk 2]
```diff
@@ -13,31 +13,38 @@
...
-	<message key="plugins.importexport.crossref.settings.description"><![CDATA[Você pode <a href="{$settingsUrl}"> configurar o plug-in de exportação/registro CrossRef aqui</a>.]]></message>
-	<message key="plugins.importexport.crossref.settings.form.description">Por favor, configure o plugin de exportação/registro CrossRef:</message>
-	<message key="plugins.importexport.crossref.settings.form.username">Nome de usuário</message>
-	<message key="plugins.importexport.crossref.settings.form.usernameRequired">Por favor, informe o nome de usuário que você revcebeu da CrossRef.</message>
-	<message key="plugins.importexport.crossref.export.unregistered">Artigos não registrados</message>
-	<message key="plugins.importexport.crossref.export.selectUnregistered">Selecionar artigos não registrados</message>
-	<message key="plugins.importexport.crossref.export.noUnregistered">Todos os artigos publicados já foram registrados (ou nenhum deles têm um DOI designado).</message>
+	<message key="plugins.importexport.crossref.description">Export or register article metadata in CrossRef format.</message>
```
Comment (asmecher) [inline plugins/importexport/crossref/locale/pt_BR/locale.xml]: This swaps the pt_BR out for en_US text.

File: plugins/metadata/dc11/locale/pt_BR/locale.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  *
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the pt_BR (Português (Brasil)) locale.
+  -->
+
+<locale name="pt_BR" full_name="Português (Brasil)">
+	<message key="plugins.metadata.dc11.articleAdapter.displayName"></message>
```
Comment (asmecher) [inline plugins/metadata/dc11/locale/pt_BR/locale.xml]: The added strings here are empty.

File: plugins/paymethod/paypal/locale/cs_CZ/locale.xml [hunk 4]
```diff
@@ -4,8 +4,8 @@
 <!--
   * plugins/paymethod/paypal/locale/cs_CZ/locale.xml
   *
-  * Copyright (c) 2014-2016 Simon Fraser University Library
-  * Copyright (c) 2006-2014 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2006-2009 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
```
Comment (asmecher) [inline plugins/paymethod/paypal/locale/cs_CZ/locale.xml]: I'm nervous about rolling back the dates for non-team-members; best to leave this as it was.

Comment (mtub) [inline plugins/paymethod/paypal/locale/cs_CZ/locale.xml]: But this is just setting the copyright dates in cs_CZ to those in every other localized version of this file.

Comment (asmecher) [inline plugins/paymethod/paypal/locale/cs_CZ/locale.xml]: Hmm, not sure where that change got in, then. OK, leave it as the PR has it.

Comment (mtub) [inline plugins/paymethod/paypal/locale/cs_CZ/locale.xml]: To clarify: no other file but this has a copyright longer than 2009 for the non-team contributors. This is what the en_US original file looks like. Please decide - I can revert this, but I don't think it's necessary.

**Comment by mtub** [comment]
everything but the copyright date issue (see above) should be resolved now.

File: plugins/paymethod/paypal/locale/cs_CZ/locale.xml (hunk 4 above)
Comment (asmecher) [inline plugins/paymethod/paypal/locale/cs_CZ/locale.xml]: I've maintained the en_US copyright dates reasonably carefully; it's most likely that this was changed by a 3rd party. It's best to stay consistent with what en_US has.

**Comment by asmecher** [comment]
Thanks, @mtub! Merged.

**Pull request merged**